src/js/reducers/cycleCountReducer.js

```diff
@@ -0,0 +1,17 @@
+import { START_COUNT } from 'actions/types';
+
+const initialState = {
+  toCount: [],
+};
+
+export default function cycleCountReducer(state = initialState, action) {
+  switch (action.type) {
+    case START_COUNT:
+      return {
+        ...state,
+        toCount: action.payload,
```

Comment:
let's map it in the same way like it is sent to the backend, so let's name it as `requests`